packages/grid/src/features/table-ui/components/table/TableHeadItem.tsx

```diff
@@ -54,14 +54,19 @@ export const TableHeadItem: React.FC<TableHeadProps> = React.memo(
       : undefined;
 
     const hasOnlyChildren = !label && !arrow && !infoIconTooltipText;
-
+    const borderTop = selected
+      ? "2px solid var(--lhds-color-blue-500)"
+      : "2px solid transparent";
     return (
       <Row
         height={"100%"}
         alignItems={"center"}
         ref={containerRef}
         overflow={overflow}
         justifyContent={alignRight ? "flex-end" : "flex-start"}
+        style={{
+          borderTop: borderTop,
+        }}
```

Comment:
Ok, first of all, we don't want `borderTop` to be blue, we want `borderBottom`.

This is not the style we're looking for:
![image](https://user-images.githubusercontent.com/6987939/213200988-b3f26ca7-f119-40c0-9c85-e31e7eb7e0fa.png)

Also, if you have a variable with the same name as the key, you could shorten it to:

```
style={{
  borderBottom,
}}
```

But I would probably just inline it like we've done with `justifyContent` above:

```suggestion
        borderBottom={`2px solid ${
          selected ? cssColor("--lhds-color-blue-500") : "transparent"
        }`}
        borderTop={"2px solid transparent"}
```
AND I would add a `borderTop` that is always transparent to cancel out the `borderBottom` so the text stays in the center of the cell.